### ISSUE TITLE
add AdminMetadata RT to Instance RT for monographs

### ISSUE THREAD

**Issue by NancyL** [LD4P/ld4ptemplates#40]
ld4p:RT:bf2:Monograph:Instance:Un-nested and ld4p:RT:bf2:Monograph:Instance

There has been a request to add the admin metadata RT to Instance RTs. This is particularly valuable for un-nested templates, where the two templates are separated, but it may be the case even with a nested template that the admin metadata differs

**Comment by NancyJean** [comment]
Yes, I'll do this in the serials unnested RTs.  I think we should add it to unnested Works too.

**Comment by NancyL** [comment]
So the question has changed somewhat. Do we want admin metadata un-nested as well? This way people could create individual templates for their institutions with defaults filled in.

**Comment by RobertRohr** [comment]
Hello,
              There’s something to be said for that approach. Besides setting defaults, one could also use an unnested template to provided admin metadata for works if we ever decide that’s desirable.

Robert

--------------------------
Robert J. Rohrbacher
Metadata Librarian for Social Sciences and Government Documents
Metadata Dept.
Stanford University Libraries
phone: 650-725-7992
fax: 650-725-1120
rrohrbac@stanford.edu

**Comment by NancyJean** [comment]
I'm ok with this for all the above reasons -- being able to apply it as need is identified to Works, Instances, and Items and to facilitate local customized templates.

**Comment by NancyL** [comment]
In dev server. Since the name RT is shared with all provision activities, role appears with all of them.
